static/css/lanyon.css

```diff
@@ -202,54 +202,64 @@ a.sidebar-nav-item:focus {
 
 /* Hide the sidebar checkbox that we toggle with `.sidebar-toggle` */
 .sidebar-checkbox {
-  display: none;
+  position: absolute;
+  opacity: 0;
+  -webkit-user-select: none;
+     -moz-user-select: none;
+          user-select: none;
 }
 
 /* Style the `label` that we use to target the `.sidebar-checkbox` */
 .sidebar-toggle {
   position: absolute;
-  top:  1rem;
+  top:  .8rem;
   left: 1rem;
   display: block;
-  width: 2.2rem;
-  padding: .5rem .65rem;
+  padding: .25rem .75rem;
   color: #505050;
   background-color: #fff;
-  border-radius: 4px;
+  border-radius: .25rem;
   cursor: pointer;
 }
+
 .sidebar-toggle:before {
-  display: block;
+  display: inline-block;
+  width: 1rem;
+  height: .75rem;
   content: "";
-  width: 100%;
-  padding-bottom: .125rem;
-  border-top: .375rem double;
-  border-bottom: .125rem solid;
-
-  /* Make the border inside the box */
-  -webkit-box-sizing: border-box;
-     -moz-box-sizing: border-box;
-          box-sizing: border-box;
+  background-image: -webkit-linear-gradient(to bottom, #555, #555 20%, #fff 20%, #fff 40%, #555 40%, #555 60%, #fff 60%, #fff 80%, #555 80%, #555 100%);
+  background-image:    -moz-linear-gradient(to bottom, #555, #555 20%, #fff 20%, #fff 40%, #555 40%, #555 60%, #fff 60%, #fff 80%, #555 80%, #555 100%);
+  background-image:     -ms-linear-gradient(to bottom, #555, #555 20%, #fff 20%, #fff 40%, #555 40%, #555 60%, #fff 60%, #fff 80%, #555 80%, #555 100%);
+  background-image:         linear-gradient(to bottom, #555, #555 20%, #fff 20%, #fff 40%, #555 40%, #555 60%, #fff 60%, #fff 80%, #555 80%, #555 100%);
 }
 
 .sidebar-toggle:active,
+#sidebar-checkbox:focus ~ .sidebar-toggle,
```

Comment:
No problem! Glad I could help.

I'll try to keep an eye on the Issues in the future and help out when I
can.

I'm using this for my personal site redesign, which happens to be on the
back burner at the moment, so thanks for putting this together and keeping
it up-to-date!



On Sat, Mar 12, 2016 at 2:14 PM, Stephen Jung <notifications@github.com>
wrote:

> In static/css/lanyon.css
> <https://github.com/tummychow/lanyon-hugo/pull/12#discussion_r55926108>:
>
> >  }
> >
> >  .sidebar-toggle:active,
> > +#sidebar-checkbox:focus ~ .sidebar-toggle,
>
> I didn't think anyone would actually take me up on the call for
> contribution, thanks!
>
> iirc lanyon only claims compat with IE9+, so I'm not concerned about IE7.
> On firefox, your suggestion looks like it works... will poke around a bit
> longer, and then maybe apply your suggestion and merge this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tummychow/lanyon-hugo/pull/12/files#r55926108>.
>